tests/integration/scripting/conftest.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+import pytest
+from typing_extensions import Protocol
+
+
+class GetScriptFixtureProtocol(Protocol):
```

Comment:
```suggestion
class GetScriptFixture(Protocol):
```

We follow this in other test files

Reply:
Done